flake.nix

```diff
@@ -67,8 +67,11 @@
             # has `md5sum`).
             export MD5SUM=md5sum
 
+            # Work around #4819, missing support for generic type aliases.
+            export GODEBUG=gotypesalias=0
```

Comment:
Why is this needed? Isn't this still disabled by default?

Reply:
The upgrade bumped Go to 1.24 which has `gotypesalias=1` as default.

Reply:
Huh, weird that I never ran into it. I'm using Go 1.24.